.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
     - php: 5.6 # PHPUnit 5.7
       env: WITH_CS=true
     - php: 7 # PHPUnit 6.x
-    - php: 7.1 # PHPUnit 6.x
+    - php: 7.1 # PHPUnit 7.x
```

Comment:
the build fails on PHP 7.2 https://travis-ci.org/giorgiosironi/eris/builds/355726627?utm_source=github_status&utm_medium=notification
but I don't see it listed here? :thinking:

Reply:
Ah wait, that build linked here is for a different commit that did contain `7.2`. Not sure how this commit status was created by Travis then.

Reply:
Did some force push? I'm going to close this and reopen a new PR with your branch since we don't want to debug Travis CI.

Reply:
Force push yes. Apparently Eris CI fails with 7.2, but then we don't need 7.2 to test PHPUnit 7.x, hence removed from this PR.

Reply:
Please reopen :)